bin/binlinks.py

```diff
@@ -9,6 +9,8 @@
 import csv
 import gzip
 
+# FIXME: why is this script here and we are not using the one in eukcc?
```

Comment:
I didn't know Paul added it to eukcc container

Reply:
it should be there.. it's in the repo at least -> https://github.com/EBI-Metagenomics/EukCC/blob/master/scripts/binlinks.py
and eukcc is on bioconda so there is biocontainer